tests/integ/modin/resample/test_resample.py

```diff
@@ -19,7 +19,8 @@
 )
 from tests.integ.utils.sql_counter import SqlCounter, sql_count_checker
 
-agg_func = pytest.mark.parametrize("agg_func", IMPLEMENTED_AGG_METHODS)
+# Parametrize on all IMPLEMENTED_AGG_METHODS except 'indices' which is tested in a separate file
+agg_func = pytest.mark.parametrize("agg_func", IMPLEMENTED_AGG_METHODS[:13])
```

Comment:
are we only skipping some specific agg functions for the test?

would it be better to filter out specific functions than relies on the first 12 (in case other update the IMPLEMENTED_AGG_METHODS later ) 

```
agg_func = pytest.mark.parametrize(
    "agg_func",
     list(filter(lambda x: x not in  ["indices"], IMPLEMENTED_AGG_METHODS))
 )
```

Reply:
Good point, done!

Reply:
Perhaps another option is to IMPLEMENTED_AGG_PROPERTIES separate from IMPLEMENTED_AGG_METHODS?

Reply:
For now since the only property currently implemented is `indices`, I'll go with Jialin's suggestion here. I don't think it matters too much here naming-wise since the code paths for other agg methods and 'indices' are quite similar.